src/Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -441,6 +439,22 @@ public void Stop()
             this.logger.LogTrace("(-):*.{0}='{1}',*.{2}='{3}'", nameof(blockValidationContext.ChainedHeader), blockValidationContext.ChainedHeader, nameof(blockValidationContext.Error), blockValidationContext.Error?.Message);
         }
 
+        /// <summary>
+        /// Calculates if coinview flush is required.
+        /// </summary>
+        /// <remarks>
+        /// For blockchains with max reorg property flush is required when consensus tip is less than max reorg blocks behind the chain tip.
+        /// If there is no max reorg property - flush is required when consensus tip timestamp is less than two weeks behind the adjusted time.
+        /// </remarks>
+        private bool FlushRequired()
+        {
+            if (this.chainState.MaxReorgLength != 0)
+                return this.Chain.Height - this.Tip.Height < this.chainState.MaxReorgLength;
+
+            int twoWeeksAsSeconds = 2 * 24 * 60 * 60;
```

Comment:
This should be a constant on the class with comment. It should be explained why this number was selected.

Reply:
Fixed